include/hipSYCL/sycl/queue.hpp

```diff
@@ -741,6 +761,71 @@ class queue : public detail::property_carrying_object
     });
   }
 
+  
+  // Explicit copy functions
+  template <typename T, int dim, access::mode mode, access::target tgt,
```

Comment:
Can we directly move to the SYCL 2020 `access_mode` alias instead of `access::mode`?

Reply:
Sure. I mainly used `access::mode` and `access::target` since the implementation in the `sycl::handler` class also uses them :).

Reply:
Yes, they are still used in some places, but I think for new code we should just use the new ones - we will have to switch everything at some point anyway :)